File: app/src/main/java/com/zygne/confetti/engine/util/Randomizer.java
package com.zygne.confetti.engine.util;

public class Randomizer {

    // Return an integer that ranges from min inclusive to max inclusive.
    public static int rndInt(int min, int max) {
        return (int) (min + Math.random() * (max - min + 1));
    }

    public static float rndDbl(float min, float max) {
        return (float) (min + (max - min) * Math.random());
    }
}

File: app/src/main/java/com/zygne/confetti/engine/util/Randomizer.kt
package com.zygne.confetti.engine.util

object Randomizer {
    fun randomInteger(min: Int, max: Int): Int {
        return (min + Math.random() * (max - min + 1)).toInt()
    }

    fun randomDouble(min: Float, max: Float): Float {
        return (min + (max - min) * Math.random()).toFloat()
    }
}